front-end/src/components/Signup/index.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import styled from 'styled-components';
+import FormInput from '../../shared/FormInput';
+import { withRouter, Link } from "react-router-dom";
+import Logo from '../../svg/Logo';
+import axios from 'axios';
+
+
+
+
+
+class Signup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      /*phoneNumber: '',
+      name: '',
+      username: '',*/
```

Comment:
Let's remove these if we don't need them anymore.

Reply:
These will be removed